uint8 command_type
float32 absolute_height
